pkg/stanza/operator/input/windows/operator.go

```diff
@@ -12,9 +12,10 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/zap"
+
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/helper"
-	"go.uber.org/zap"
```

Comment:
please revert

Reply:
make fmt  removed add it on top. Anyway, I revert.

Reply:
haha :)
`make gci` does the right job to deal with imports